app/multiples.hs

```diff
@@ -7,9 +7,14 @@ checkmul x = if ( x `mod` 3) == 0
 
 tolist x = [x]
 
-findmul :: [Int] -> [Int]
-findmul [] = [0]
-findmul xs = (tolist (checkmul (head xs))) ++ (findmul (tail xs))
+findmul :: Integer -> Integer
+--findmul xs = (tolist (checkmul (head xs))) ++ (findmul (tail xs))
+findmul n 
+        |n < 10 = if (checkmul n) == n
```

Comment:
what is the purpose of this `n <  10` condition. Also this solution is incorrect.